examples/example.tf

```diff
@@ -0,0 +1,42 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+locals {
+  publicly_accessible = true
+}
+
+module "vpc" {
+  source                             = "terraform-aws-modules/vpc/aws"
+  version                            = "2.24.0"
+  name                               = "example-vpc"
+  cidr                               = "10.99.0.0/19"
+  azs                                = ["eu-west-1a", "eu-west-1b"]
+  public_subnets                     = ["10.99.9.0/24", "10.99.10.0/24"]
+  redshift_subnets                   = ["10.99.11.0/24", "10.99.12.0/24"]
+  create_database_subnet_route_table = false
+  create_redshift_subnet_group       = false
```

Comment:
Uuh... Let's use our own VPC modules in our own examples instead 😄

Reply:
As discussed, VPC module left out of the example. Required module vars speak for themselves